content/zh/docs/concepts/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: OpenTelemetry 概念 
+linkTitle: Concepts
```

Comment:
if you also translate the linkTitle it will also be updated in the navigation

Reply:
ok, translated.